src/v2/Apps/Fair/Components/FairExhibitors/FairExhibitorCard.tsx

```diff
@@ -62,75 +54,51 @@ export const FairExhibitorCard: React.FC<FairExhibitorCardProps> = ({
   }
 
   return (
-    <Box>
-      <RouterLink
-        to={partner.href}
-        noUnderline
-        onClick={() => tracking.trackEvent(tappedPartnerTrackingData)}
-      >
-        <Flex mb={1} flex={1}>
-          <BorderBox width={52} height={52} p={0} mr={1}>
-            {profile?.icon?.cropped && (
-              <Image
-                lazyLoad
-                src={profile?.icon?.cropped?.src}
-                srcSet={profile?.icon?.cropped?.srcSet}
-                alt={`Logo of ${name}`}
-                width={50}
-                height={50}
-              />
-            )}
-          </BorderBox>
-          <Box overflow="hidden">
-            <Text variant="md" overflowEllipsis>
-              {name}
+    <RouterLink
+      to={partner.href}
+      noUnderline
+      onClick={() => tracking.trackEvent(tappedPartnerTrackingData)}
+    >
+      <Flex mb={1} flex={1}>
+        <BorderBox width={52} height={52} p={0} mr={1}>
+          {profile?.icon?.cropped && (
+            <Image
+              lazyLoad
+              src={profile?.icon?.cropped?.src}
+              srcSet={profile?.icon?.cropped?.srcSet}
+              alt={`Logo of ${name}`}
+              width={50}
+              height={50}
+            />
+          )}
+        </BorderBox>
+        <Box>
+          <Text variant="md" overflow="clip">
+            {name}
+          </Text>
+          {cities?.length ? (
+            <Text variant="md" color="black60" overflow="clip">
+              {partnerAddress([...cities])}
```

Comment:
**question**: Does `partnerAddress([...cities])` have a different effect than `partnerAddress(cities)`?

Reply:
Yes, because `cities` are read only, so to avoid linter errors we need to create new array

Reply:
Ahh, that makes sense. I see that linter error now:

![Screen Shot 2021-09-02 at 1 45 54 PM](https://user-images.githubusercontent.com/123595/131838185-d3a37709-9268-4aa1-82e0-94beea42799e.png)

In that case, would it be better to update the argument types to also expect a readonly array?

```typescript
const partnerAddress = (cities: readonly (string | null)[]) => {
  const visibleCities = cities?.slice(0, VISIBLE_CITIES_NUM).join(", ")

  if (cities.length > VISIBLE_CITIES_NUM) {
    return `${visibleCities}, +${cities.length - VISIBLE_CITIES_NUM} more`
  }

  return visibleCities
}
```

It looks like the `partnerAddress` already creates a copy right away with `#slice` and doesn't mutate the array afterwards.